bouncerscript/test/test_work_dir/task.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Nit: It looks like this file is the same as `task_submission.json`. Should we keep both?

Reply:
https://github.com/mozilla-releng/scriptworker/blob/master/scriptworker/client.py#L42 expects `task.json` so unless we mock this function and overwrite the filename, the unit testing for `async_main` will barf. If we kill this, all the functions using any of the `submission_context` or `alias_context` will barf if we remote the others as they are loading the configurations.

I guess I can redo that part and make it nicer, but let's not block on that for now. I raise https://github.com/mozilla-releng/bouncerscript/issues/13 to track that separately.

Reply:
Okay, I'll continue the conversation there :)